jjb/jobs/rho-copr-install.yaml

```diff
@@ -1,6 +1,6 @@
 - job-template:
     name: 'rho-copr-install-{os}'
-    node: '{os}-np'
+    node: '{os}-os'
```

Comment:
Nitpick: Maybe choose a different values for the template variable so it is different than "os" standing for "openstack"?
Like `distro`?

Reply:
I see what you mean, yeah I felt the same about having it named as os but didn't had a better idea on how to name. `distro` is a good name.